src/CursorHandler.js

```diff
@@ -1,15 +1,37 @@
 export default function CursorHandler () {
-    function updateCursor(id) {
-        var url = 'url(/img/' + id + '.png), auto';
-        $("#bgimg").css('cursor', url);
-        $("#mei_output").css('cursor', url);
+    // this.origHeight = origHeight;
+    // this.origWidth = origWidth;
+
+    function updateCursor() {
+        $("#bgimg").css('cursor', 'crosshair');
```

Comment:
Would it be possible to just apply it to the container of the background image and MEI rather than applying it to each?

Reply:
Also since `CursorHandler` doesn't store anything and is just a set of functions we might as well just define its functions like `CursorHandler.updateCursor` instead of `CursorHandler.prototype.updateCursor` and not bother instantiating `CursorHandler` objects.

Reply:
@JRegimbal For the container, it seems like the container is larger than just the background image and the MEI output so if you click on the whitespace surrounding the image, the neumes are not unselected. That's why I did them individually. As for the cursorHandler, sounds good to me!